src/types/select.js

```diff
@@ -25,7 +27,9 @@ export default  ngModule => {
           optionsAttr: check.string.optional,
           labelProp: check.string.optional,
           valueProp: check.string.optional,
-          groupProp: check.string.optional
+          groupProp: check.string.optional,
+          notNull: check.boolean.optional,
```

Comment:
using `api-check` version `7.5.5` throws next error:

```
TypeError: Cannot read property 'optional' of undefined
```

This is because it should be used `check.bool.optional` instead of `check.boolean.optional`

please correct this error.